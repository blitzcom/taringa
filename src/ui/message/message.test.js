import renderer from 'react-test-renderer';

import Message from './message';

describe('ui/message', () => {
  def('children', () => 'My message');
  def('subject', () => renderer.create(<Message>{$children}</Message>).toJSON());

  it('renders message with children as message', () => {
    expect($subject).toMatchSnapshot();
  });
});
