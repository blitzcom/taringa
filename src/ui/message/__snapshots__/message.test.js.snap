// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/message renders message with children as message 1`] = `
<div
  className="wrapper"
>
  My message
</div>
`;
